src/main/java/duke/commands/Command.java

```diff
@@ -0,0 +1,21 @@
+package duke.commands;
+
+import duke.exceptions.UserInputError;
+import duke.tasks.TaskList;
+import duke.util.Storage;
+import duke.util.Ui;
+
+/**
+ * Command class that represents each user command.
+ */
+public abstract class Command {
+    /**
+     * Executes the user's command.
+     * @param tasks Current tasklist.
+     * @param ui User interface of Duke.
```

Comment:
I think you should include an empty line between the description and parameter section based on [this](https://se-education.org/guides/conventions/java/intermediate.html#comments).

```suggestion
     * Executes the user's command.
     * 
     * @param tasks Current tasklist.
     * @param ui User interface of Duke.
```